{
	"id": "CVE-2020-22570",
	"sourceIdentifier": "cve@mitre.org",
	"vulnStatus": "Analyzed",
	"published": "2023-08-22T19:16:19.297",
	"lastModified": "2023-08-25T19:06:16.767",
	"descriptions": [
		{
			"lang": "en",
			"value": "Memcached 1.6.0 before 1.6.3 allows remote attackers to cause a denial of service (daemon crash) via a crafted meta command."
		}
	],
	"references": [
		{
			"source": "cve@mitre.org",
			"tags": [
				"Exploit",
				"Issue Tracking"
			],
			"url": "https://github.com/memcached/memcached/issues/636"
		}
	],
	"metrics": {
		"cvssMetricV31": [
			{
				"source": "nvd@nist.gov",
				"type": "Primary",
				"cvssData": {
					"version": "3.1",
					"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
					"attackVector": "NETWORK",
					"attackComplexity": "LOW",
					"privilegesRequired": "NONE",
					"userInteraction": "NONE",
					"scope": "UNCHANGED",
					"confidentialityImpact": "NONE",
					"integrityImpact": "NONE",
					"availabilityImpact": "HIGH",
					"baseScore": 7.5,
					"baseSeverity": "HIGH"
				},
				"exploitabilityScore": 3.9,
				"impactScore": 3.6
			}
		]
	},
	"weaknesses": [
		{
			"source": "nvd@nist.gov",
			"type": "Primary",
			"description": [
				{
					"lang": "en",
					"value": "CWE-77"
				}
			]
		}
	],
	"configurations": [
		{
			"nodes": [
				{
					"operator": "OR",
					"cpeMatch": [
						{
							"vulnerable": true,
							"criteria": "cpe:2.3:a:memcached:memcached:*:*:*:*:*:*:*:*",
							"matchCriteriaId": "9CDEE25B-32D1-4097-BDAB-3C5673C6CE5B",
							"versionStartIncluding": "1.6.0",
							"versionEndExcluding": "1.6.3"
						}
					]
				}
			]
		}
	]
}
